src/quo2/components/community/community_list_view.cljs

```diff
@@ -118,7 +118,8 @@
      [rn/view
       {:flex-direction :row
        :border-radius  16
-       :align-items    :center}
```

Comment:
consider making a community-list folder and in there a view.cljs and style.cljs file and pout these styles in the style file